app/models/organization/handle.rb

```diff
@@ -0,0 +1,37 @@
+class Organization::Handle
+  RESTFUL_ACTIONS = %w[
+    create
+    destroy
+    edit
+    index
+    new
+    show
+    update
+  ].freeze
+
+  ROUTES = %w[
+    admin
+    api
+    dashboard
+    gems
+    invitation
+    invitations
+    members
+    memberships
+    onboarding
+    profile
+    settings
+    stats
+    teams
+    users
```

Comment:
This list is significantly pared down from what it was. It can be pared down further, but I don't think we're at risk of organizations being created from gems of these names. `gems` is by far the most popular and owned by us. Most of the other names here are pre-v1 versions from the 2010s.